x-pack/plugins/maps/public/classes/layers/vector_layer/utils.tsx

```diff
@@ -77,15 +77,21 @@ export async function syncVectorSource({
   } = syncContext;
   const dataRequestId = SOURCE_DATA_REQUEST_ID;
   const requestToken = Symbol(`${layerId}-${dataRequestId}`);
-  const canSkipFetch = syncContext.forceRefresh
-    ? false
-    : await canSkipSourceUpdate({
-        source,
-        prevDataRequest,
-        nextMeta: requestMeta,
-        extentAware: source.isFilterByMapBounds(),
-        getUpdateDueToTimeslice,
-      });
+
+  let canSkipFetch = false;
```

Comment:
Can you keep the original structure and use a const and optional chaining operator, instead of changing to `if/else`?